assets/espresso/simple.j2.in

```diff
@@ -0,0 +1,16 @@
+&inputsimple
+    prefix = '__prefix__'
+    outdir = {% raw %}'{{ JOB_WORK_DIR }}/outdir'{% endraw %}
+    calc_mode=0
+    num_nbndv=16
+    num_val=16
+    num_cond=24
+    s_bands=0.1
+    s_product=1.0
```

Comment:
Maybe we could use https://github.com/cphyc/fortran-linter and pre-commit hook to format this

Reply:
Looks like the linter works, except for the 4 spaces. Let's use the fork in https://github.com/Exabyte-io/fortran-linter